### ISSUE TITLE
#quit with a line feed after before ;; produces a mangled output in fsi

### ISSUE THREAD

**Issue by 7sharp9** [Microsoft/visualfsharp#5453]
Please provide a succinct description of the issue.

#### Repro steps

```
Microsoft (R) F# Interactive version 10.1.0 for F# 4.1
Copyright (c) Microsoft Corporation. All Rights Reserved.

For help type #help;;

> #quit
;;
Invalid directive '#qyuit '
```

#### Expected behavior

fsi exists as normal

#### Actual behavior

As above

#### Known workarounds

Use `#quit;;`

#### Related information 

* Operating system OSX
* dotnet/sdk/2.1.301

**Comment by nosami** [comment]
Are you running via `fsharpi` ?

This is what I see

```
12:51 $ fsharpi

Microsoft (R) F# Interactive version 10.1.0 for F# 4.1
Copyright (c) Microsoft Corporation. All Rights Reserved.

For help type #help;;

> #quit
- ;;

- Exit...
```

**Comment by 7sharp9** [comment]
That with monos fsi, this is the one in the sdk:

```
dotnet /usr/local/share/dotnet/sdk/2.1.301/FSharp/fsi.exe
```

**Comment by nosami** [comment]
Working for me with 2.1.302

```
13:08 $ dotnet /usr/local/share/dotnet/sdk/2.1.302/FSharp/fsi.exe

Microsoft (R) F# Interactive version 10.1.0 for F# 4.1
Copyright (c) Microsoft Corporation. All Rights Reserved.

For help type #help;;

> #quit
;;
✔ ~/src/md-addins [openjdk-updater L|●1✚ 10…5⚑ 2]
```

**Comment by 7sharp9** [comment]
Hmm, ill download and try it, weird one that.

**Comment by 7sharp9** [comment]
Yep fixed in 2.1.302, I didn't even know that was out :-)
